tests/python/gaia-ui-tests/gaiatest/mixins/gaia.py

```diff
@@ -19,6 +19,11 @@ def __init__(self, **kwargs):
                          default=False,
                          help='restart target instance between tests')
 
+        group.add_option('--locale',
+                         default="undefined",
```

Comment:
Okay, let's show the in the comment that the default language is the one set on the device, then.